tests/jetstream.rs

```diff
@@ -516,3 +517,101 @@ fn jetstream_ordered() {
         assert_eq!(message.data, (i as i64).to_be_bytes());
     }
 }
+
+#[test]
+fn jetstream_pull_subscribe_batch() {
+    let s = util::run_server("tests/configs/jetstream.conf");
+    let nc = nats::Options::new()
+        .error_callback(|err| println!("error!: {}", err))
+        .connect(&s.client_url())
+        .unwrap();
+    let js = nats::jetstream::new(nc);
+
+    js.add_stream(&StreamConfig {
+        name: "TEST".to_string(),
```

Comment:
`add_stream` takes in `StreamConfig` struct, which for simplicity reasons owns values  (which avoids lot of fighting borrow checked on a struct that borrows) so it uses `String` instead of `&str`.

If we want to have a nicer interface (though I think that Rustaceans are used to that 🙂 ) I would add  some utilities for creating `StreamConfig` with builder, or have a builder for `add_stream` that could take in both `String` and `&str`. Tho I would do it in another PR, as that can be applied to most of `JetStream` management API.

That part of API was not touched by recent refactors, so some care here can improve things.

Reply:
Be nice to be able to use bare strings "TEST"

Reply:
Ok, will do that as follow-up to this PR.

Reply:
https://github.com/nats-io/nats.rs/issues/303